interface/src/app/gene/network/network.less

```diff
@@ -62,3 +96,8 @@ text.legend-title {
 #gene-slider .rz-selection {
   background: magenta;
 }
+
+#download-menu {
```

Comment:
It's usually not ideal to use id's for CSS styling because it can lead to specificity wars:
https://stuffandnonsense.co.uk/archives/css_specificity_wars.html (an old article but still holds true...and you can't beat star wars metaphors 😉)

Reply:
I'm aware of that, but didn't have a better idea here. Suggestions?

We're already using IDs for the `#gene-slider` and `#edge-slider`, so we can't fully eliminate the technique unless we solve it for those cases also. You probably know this, but keep in mind that the `.less` we write gets concatenated into one large `.css` file during our build, so we need enough specificity that we don't inadvertently target unrelated components and views.

We definitely need better structure in our `.less` organization with this project, so I very much welcome your advice.

Reply:
My favorite approach for reducing specificity but still making sure styles are targeting the correct components is to use [BEM](http://getbem.com/) 😍 . Its a very straightforward naming convention for modular CSS.

But obviously, that's a larger fix, so this isn't a blocker for approving this PR. Just something to think about :)

Reply:
Excellent. I have seen BEM before, but not sure how I came across it. Maybe Deepa? It's good to have a way forward for better CSS structure.

Anyway, in this context, I think the part of that methodology that applies is the concept of the "block". You are right that we can't fix the whole codebase now, but I'd like to not actually make it worse. I'm going to switch to using `.class` here to take a step in the right direction.

We're really in this mess because nobody sticks to writing semantic HTML (bootstrap is a major offender, IMO) so we have to use CSS in ways it wasn't meant for to make up for that... but that's an even *bigger* problem than the CSS in this codebase.